pages/about.html

```diff
@@ -0,0 +1,129 @@
+                "Not personal info outlet" and The shield is protecting the cage from the outside world.
+            </p>
+
+            <p class="desktop-extra">
+                The logo competition, a celebration of creativity and innovation, concluded on May 20th this year. We
+                were thrilled
+                to witness the exceptional talent and ingenuity displayed by aspiring graphic designers. Amongst the
+                outstanding
+                entries, Peter Kibet's submission stood out, captivating hearts with its remarkable concept and
+                exquisite execution.
+                Peter's artistic prowess serves as an inspiration to young graphic designers, reminding them of the
+                limitless
+                possibilities that lie within their imagination. Embrace your unique vision, unleash your creativity,
+                and let your
+                designs paint a vibrant world of inspiration and storytelling. The future of graphic design rests in
+                your hands!
+            </p>
+
+
+            <div class="company-img">
+                <img src="./pageAssets/DefenderCage Logo.png" alt="">
+            </div>
+
+            <div class="previous-summits">
+                <h3>See the previous OCD Summits</h3>
+                <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+                <ul>
+                    <li>
+                        <a href="https://summit.creativecommons.org/2018/">
+                            <span>2021 Summit</span>
+                            <p>OCD Summit 2018 in Moscow, Russia.</p>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://summit.creativecommons.org/2019/">
+                            <span>2022 Summit</span>
+                            <p>OCD Summit 2019 in Lisbon, Portugal.</p>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+            <div id="partners">
+                <h3>Partners</h3>
+                <div class="contributors">
+                    <img src="./pageAssets/airbnb 1.png" alt="">
+                    <img src="./pageAssets/daumkakaologo 1.png" alt="">
+                    <img src="./pageAssets/googlekorea 1.png" alt="">
+                    <img src="./pageAssets/mozilla 1.png" alt="">
+                    <img src="./pageAssets/naver 1.png" alt="">
+                </div>
+            </div>
+        </section>
+
+    </main>
+
+    <footer>
+        <div class="footer-section about">
+            <h1 class="logo-text"><span>©Optimus Cage Defender</span></h1>
+            <p>
+                Optimus Cage Defender is a non-profit organization that aims to provide a secure digital
+                environment for everyone.
+            </p>
+    </footer>
+
```

Comment:
- [ ] Please make sure your footer follows the design layout that is, It contains a logo aligned to the left and text aligned to the right.

|Cindy's design|Your design|
|----|----|
|<img width="762" alt="Screenshot 2023-05-25 at 1 01 56 PM" src="https://github.com/Peter-Kibet/conferencePage/assets/44798044/dea7c0ed-b07b-4275-a0e9-7849a5ec92d3">|<img width="1275" alt="Screenshot 2023-05-25 at 1 00 48 PM" src="https://github.com/Peter-Kibet/conferencePage/assets/44798044/25857d66-be5a-44f5-bd5a-75bb10b7db99">|